track.go

```diff
@@ -209,21 +209,30 @@ func (track *baseTrack) bind(ctx webrtc.TrackLocalContext, specializedTrack Trac
 }
 
 func (track *baseTrack) unbind(ctx webrtc.TrackLocalContext) error {
-	track.mu.Lock()
-	defer track.mu.Unlock()
-
-	ch, ok := track.activePeerConnections[ctx.ID()]
-	if !ok {
-		return errNotFoundPeerConnection
+	ch, err := track.removeActivePeerConnection(ctx.ID())
+	if err != err {
+		return err
 	}
 
 	doneCh := make(chan struct{})
 	ch <- doneCh
```

Comment:
Just a small question why are you doing this?
If you want to close a channel then you can do close(ch) instead.

Reply:
https://github.com/pion/mediadevices/blob/b3c94a1f7bc2a1ef43c4056258269083a7e760f5/track.go#L174-L192
`doneCh` is sent to a routine created by `bind()` through `ch` and the routine closes the `doneCh` after finishing the loop.
`<-doneCh` waits until finishing the loop.

Reply:
Yep. Makes sense.